common/views/components/IIIFPresentationPreview/IIIFPresentationPreview.js

```diff
@@ -241,17 +241,18 @@ const IIIFPresentationDisplay = ({
   const [imageTotal, setImageTotal] = useState(null);
   const iiifPresentationManifest = useContext(ManifestContext);
   const fetchThumbnails = async () => {
-    const manifestData = iiifPresentationManifest;
-    try {
-      setImageTotal(getCanvases(manifestData).length);
-      setImageThumbnails(
-        previewThumbnails(
-          manifestData,
-          orderedStructuredImages(structuredImages(manifestData)),
-          4
-        )
-      );
-    } catch (e) {}
+    if (iiifPresentationManifest) {
+      try {
```

Comment:
Is there a reason for this `try` still as there is no async work here?

Reply:
nope, I'll remove